packages/augur-ui/src/modules/market/components/market-view/betting-market-view.tsx

```diff
@@ -58,26 +85,49 @@ const BettingMarketView = ({ location }: BettingMarketViewProps) => {
           />
         </div>
       </div>
+      <OutcomeGroup
+        orderBook={orderBook}
```

Comment:
don't pass orderBook -- OutcomeGroup can just `useMarketsStore()` to get `orderbooks`, just use `marketId` so you can get the right book. also don't pass theme, same reason.